### PR TITLE
Added contribution guidelines and development aids to readme

### LINKED ISSUE #40: Add a howto run / howto dev on the readme.md
How to deploy...

How to setup your dev environment ...

### PR DESCRIPTION
**Pull request by CptPie**
Not much more to say - closes #40

### CODE REVIEW

**Comment by dbaudisch** [comment]
The contributing guidlines should be extracted into it'S own CONTRIBUTING.md.
https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

**Review by dbaudisch — CHANGES_REQUESTED** [review]
The contributing guidlines should be extracted into their own CONTRIBUTING.md file.
https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

**Review by dbaudisch — APPROVED** [review]

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Contributing to the development
+The second way to contribute is to implement/fix specific issues yourself and posting a pull request to this project.
+
+### Getting started with development
+To get your development environment started make sure to have golang installed and up to date.
+After forking this repository and cloning your fork change into the `MoviePolls` folder and you will find the `Makefile` of this project.
+To build the project just execute the `Makefile` with `make`. A new folder `bin` will be created with an executable file called `server`.
+
+Before executing the resulting file you have to create the folder `MoviePolls/db` and within this new folder an empty json file `data.json` (the file is not completely empty but contains `{}`). If you do not create that file beforehand the server will not start.
```
Comment (zorchenhimer) [inline contributing.md:22]: This is incorrect.  Just the folder needs to exist, the json file will get created if it's not there (this folder should probably be added to the `Makefile` in a separate commit).

Comment (CptPie) [reply contributing.md:22]: Alright, my bad - i was fairly sure that the json file was necessary

**Pull request merged**